Essentials/src/main/resources/messages.properties

```diff
@@ -1010,6 +1011,12 @@ removeCommandUsage1Description=Removes all of the given mob type in the current
 removeCommandUsage2=/<command> <mob type> <radius> [world]
 removeCommandUsage2Description=Removes the given mob type within the given radius in the current world or another one if specified
 removed=\u00a76Removed\u00a7c {0} \u00a76entities.
+renamehomeCommandDescription=Rename one of your homes.
```

Comment:
```suggestion
renamehomeCommandDescription=Rename a home.
```

Reply:
Or `Renames`, whichever is more consistent with other lang strings